### ISSUE TITLE
Add edit button on variable form drawer in read view

### ISSUE THREAD

**Issue by Gladorme** [perses/perses#1390]
### Proposal

Add edit button on variable form drawer in read view

**Comment by celian-garcia** [comment]
Absolutely, more generally, there is probably a need to reconciliate the Variable and Datasource CRUD, taking the best of each other.

**Comment by AntoineThebaud** [comment]
After discussion we decided to somehow merge the 2 designs, as both make sense.
